app/models/user.rb

```diff
@@ -506,7 +506,7 @@ def staff?
   end
 
   def admin?
-    Rails.env.production? && EnvConfig.WCA_LIVE_SITE? ? software_team_admin? : software_team?
+    Rails.env.production? && EnvConfig.WCA_LIVE_SITE? ? software_team_admin? : (software_team? || software_team_admin?)
```

Comment:
This is done because wst_admin_role is not considered as a subset of wst_role, and technically, one person can be part of WST admin and not WST.